app/views/releases/_release.html.erb

```diff
@@ -10,9 +10,11 @@
     <% end %>
   </td>
   <td>
-    <% release.changeset.authors.take(8).each do |user| %>
-      <%= github_user_avatar(user) %>
-    <% end %>
+    <span style="white-space: nowrap">
+      <% release.changeset.authors.take(8).each do |user| %>
```

Comment:
should we uniq this to remove clutter ?

Reply:
I think `authors` already has a `uniq` call